config/settings.yml.sample

```diff
@@ -1,4 +1,8 @@
 ---
+# User under the app will run, default is 'pi' for a raspberry
+running_under_user: pi
+running_under_group: pi
```

Comment:
Why don't we default these to the user that executes ansible?

Reply:
same as the other comment, I guess we can default them if we assume running under the final user, that would simplify a bit the configuration

Reply:
Well I think we are assuming that people are doing that at the moment with the pi user. So I think it's probably a good default to use, and then people who want to do more exotic things can set the var or invoke ansible themselves etc?

Reply:
thinking a bit more about that (and the two other comments) I'm not sure it would work with another user than the one it's ran under, mostly because the scripts aren't moving and are "expected" to be under the current user.

I think we can move thoses two out of settings.yml, and use two facts on the group vars defaults file:
```
# from an `ansible -m setup localhost`
        "ansible_user_dir": "/home/dashie",
        "ansible_user_id": "dashie",
```

Reply:
Sounds good, I would kinda like to make this run as a proper ansible role one day, but we can cross that bridge when we come to it :+1:

Reply:
great, I will change that part a bit later today